raaghu-components/src/rds-comp-email-settings/rds-comp-email-settings.tsx

```
@@ -131,9 +146,10 @@ const RdsCompEmailSettings = (props: RdsCompEmailSettingsProps) => {
                         placeholder="Enter Email Address"
                         customClasses="form-control"
                         dataTestId='address'
-                        validatonPattern={
-                            /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
-                            }
+                        inputType="email"
```

ESLint

[error] Expected indentation of 32 spaces but found 24.

(indent)